packages/framework/fluid-framework/api-extractor.json

```diff
@@ -4,7 +4,7 @@
 	"bundledPackages": ["@fluidframework/*"],
 	"messages": {
 		"extractorMessageReporting": {
-			// TODO: Add missing documentation and remove this rule override
+			// TODO: Fix violations and set logLevel to "error"
```

Comment:
The base config currently has this disabled, so the comment here wasn't quite right